src/portal-impl.c

```diff
@@ -587,19 +587,24 @@ find_portal_implementation (const char *interface)
         }
     }
 
-#if 0
```

Comment:
The `#if 0`'d implementation could be kept as `#if 0` below this fallback, if preferred.

Reply:
No, I think we can just drop it.